src/diff.c

```diff
@@ -1863,6 +1872,16 @@ int git_diff_commit_as_email(
 	opts.summary = git_commit_summary(commit);
 	opts.author = git_commit_author(commit);
 
+	/* The first line of the commit message is identical to
+	 * the summary and thus needs to be skipped */
+	if ((ptr = git_commit_message(commit)) != NULL) {
+		/* Find end of first line */
+		ptr = strpbrk(ptr, "\r\n");
```

Comment:
However, note that what we want to find is the end of the first paragraph, not the first line. That is, we're looking for the first `"\n\n"` (or, I suppose maybe also `"\r\n\r\n"`, though I don't know if any editor will actually use CRLF for the commit message) as a separator between the subject and the body.